types/pinyin/index.d.ts

```diff
@@ -0,0 +1,24 @@
+// Type definitions for pinyin 2.8.3
+// Project: https://github.com/hotoo/pinyin
+// Definitions by: AnJun Wang <https://github.com/wanganjun>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = pinyin;
+
+declare function pinyin(words: any, options?: Options): Array<Array<string>>;
+
+declare namespace pinyin {
+    export function compare(a: any, b: any): -1 | 0 | 1;
+    export const STYLE_NORMAL: number;
+    export const STYLE_TONE: number;
+    export const STYLE_TONE2: number;
+    export const STYLE_TO3NE: number;
+    export const STYLE_INITIALS: number;
+    export const STYLE_FIRST_LETTER: number;
```

Comment:
It seems TS can do `namespace + enum` and `function + namespace`, but it somehow can't do `function + enum`, just ignore this suggestion, sorry for the trouble.